packages/files-ui/src/Components/Modules/Settings/SubscriptionTab/ChangePlan/PaymentMethodSelector.tsx

```diff
@@ -157,14 +160,14 @@ const PlanDetails = ({ goBack, onSelectPaymentMethod }: IPaymentMethodProps) =>
         <div className={classes.buttons}>
           <Button
             onClick={goBack}
-            variant="secondary"
+            variant="text"
           >
-            <Trans>Go back</Trans>
+            <Trans>Go Back</Trans>
```

Comment:
that's the only one with a big B otherwise, I'd say we should change them all or nothing.
```suggestion
            <Trans>Go back</Trans>
```